hugo/content/_index.html

```diff
@@ -0,0 +1,155 @@
+            <img src="/assets/sprotty-xtext.png" alt="" class="mx-[-50px] mt-3 z-30" />
+            <img src="/assets/sprotty-dependency.png" alt="" class="mb-3 z-20" />
+        </div>
+        <p class="text-sprottyTeal text-xl text-center pb-20 font-medium">
+            Sprotty is an open-source project enabling the creation of powerful diagramming tools and graphical editors.
+            <br><br>
+            The project is maintained by <a href="https://www.typefox.io" class="underline decoration-dashed">TypeFox</a> and <a href="https://eclipsesource.com/" class="underline decoration-dashed">EclipseSource</a>, offering professional support to elevate your project to new heights!
+        </p>
+        <div class="flex flex-row gap-10 justify-center items-center pb-10">
+            <a href="/docs/getting_started" class="bg-sprottyMidBlue bg-opacity-50 rounded-xl px-8 py-3 text-sprottyDarkBlue shadow-md">Get started</a>
+            <div class="flex flex-row gap-2">
+                <a href="#what-is-sprotty" class="text-sprottyDarkBlue">Learn more</a>
+                <img src="/assets/chevron-down.svg" alt="">
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="what-is-sprotty">
+    <div class="flex flex-row bg-waves bg-[length:auto_100%] bg-center bg-repeat-x px-10 pt-10 gap-5">
+        <div class="basis-1/2">
+            <p class="text-sprottyDarkBlue font-montserrat pb-20">
+                Sprotty provides a framework for rendering diagrams using the <span class="font-semibold">Scalable Vector Graphics</span> (SVG) format, pluggable to powerful layout engines such as the <a href="https://www.eclipse.org/elk/" class="font-semibold underline decoration-dashed">Eclipse Layout Kernel</a> (ELK).
+                <br><br>
+                Sprotty is <span class="font-semibold"> fully customizable </span>, from the look of your diagram, down to its behavior.
+                <br><br>
+                The example diagram depicts electronic components and their relationship for an electronically controlled vehicle (from <a href="https://www.etas.com/en/products/ehandbook.php" class="underline decoration-dashed">ETAS EHANDBOOK</a>).
+                <br><br>
+                Try it live! Start exploring the diagram, hover over components, click on them, discover what Sprotty can do!
+            </p>
+        </div>
+        <div id="sprotty-ascet-example" class="bg-white basis-1/2 mt-[-4rem] shadow-md"></div>
+    </div>
+</section>
+
+<section id="characteristics">
+    <div class="flex flex-row gap-5 bg-im px-10 mb-[-3rem]">
+        <div class="basis-1/2">
+            <img src="/assets/sprotty-dependency.png" alt="" class="mt-[-4rem]"/>
+            <img src="/assets/sprotty-multicore.png" alt="" class="mt-[-5rem] pl-36 z-10 relative"/>
+            <img src="/assets/sprotty-yang.png" alt="" class="mt-[-5rem]"/>
+            <img src="/assets/sprotty-xtext.png" alt="" class="mt-[-5rem] pl-36"/>
```

Comment:
Please use higher-resolution versions of these screenshots (see [README](https://github.com/eclipse-sprotty/sprotty#readme)).